MSBot/src/models/luisService.ts

```diff
@@ -16,11 +15,7 @@ export class LuisService extends ConnectedService implements ILuisService {
     }
 
     public toJSON(): Partial<ILuisService> {
```

Comment:
>Partial [](start = 21, length = 7)

We should get rid of the Partial<> That will use TypeSCript to force all callers to pass fully populated object